sdk/go/kv/kv.go

```diff
@@ -0,0 +1,137 @@
+// Package kv provides access to key value stores within Spin
```

Comment:
Do we want to start only building these when building for WASI and otherwise provide a no-op stub? It'll make building mixed spin/non-spin codebases easier.

Reply:
Seems fine in theory. Would you copy all the function stubs or is there something to automate that?